website/src/main/tut/docs/rules/NoFinalize.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: docs
+title: NoFinalize
+---
+
+** syntactic **
```

Comment:
Is this a convention we use elsewhere too?

Reply:
I just introduced it. I think since those rules/rewrite does not require compilation we should tag them.

Reply:
Makes sense. Maybe we can refine this to make it look like a tag, but definitely in a future PR.

Reply:
We should be able to generate all docs for rules automatically, tables with the configuration options, syntactic/semantic, caveats, example, since version number, etc.